docker/build/conf/dolphinscheduler/worker.properties.tpl

```diff
@@ -21,6 +21,9 @@ worker.exec.threads=${WORKER_EXEC_THREADS}
 # worker heartbeat interval
 worker.heartbeat.interval=${WORKER_HEARTBEAT_INTERVAL}
 
+# worker tenant auto create
+worker.tenant.auto.create=true
+
```

Comment:
The default value of false may be better. WDYT?

Reply:
In `worker.properties`, the default value of `worker.tenant.auto.create` is `false`. But in the file `docker/build/conf/dolphinscheduler/worker.properties.tpl` for container, set it to true will be better

Reply:
In `worker.properties`
![image](https://user-images.githubusercontent.com/4902714/111581300-f0b30980-87f3-11eb-8a36-58378f8820ef.png)

In `docker/build/conf/dolphinscheduler/worker.properties.tpl`
![image](https://user-images.githubusercontent.com/4902714/111581343-0a545100-87f4-11eb-964e-8aad5b436b68.png)